release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -695,6 +695,11 @@ Node disruption policies for Google Cloud Platform (GCP) clusters has been promo
 [id="ocp-4-17-machine-management_{context}"]
 === Machine management
 
+[id="ocp-4-17-enhancements-aws-mapi-placement_{context}"]
+==== Supporting AWS Placement Group Partition Number in CAPI & MAPI
+
+The {product-title} release {product-version} introduces the `placementGroupPartition` field for {product-title} `MachineSets` on AWS. This feature allows users to specify a partition number within an existing placement group, enabling precise instance allocation and improved fault tolerance. For more information, see xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc#machineset-aws-existing-placement-group_creating-machineset-aws[Assigning machines to placement groups for Elastic Fabric Adapter instances by using machine sets].
```

Comment:
- `MachineSet` is the correct term. https://github.com/openshift/api/blob/3c700b6cb32b58a619428e246b4ad5040565ac13/machine/v1beta1/types_machineset.go#L28



```suggestion
The {product-title} release {product-version} introduces the `placementGroupPartition` field for {product-title} `MachineSet` on AWS. This feature allows users to specify a partition number within an existing placement group, enabling precise instance allocation and improved fault tolerance. For example, see xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc#machineset-aws-existing-placement-group_creating-machineset-aws[Assigning machines to placement groups for Elastic Fabric Adapter instances by using machine sets].
```